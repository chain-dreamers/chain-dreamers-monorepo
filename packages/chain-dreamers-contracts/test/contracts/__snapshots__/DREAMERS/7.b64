data:application/json;base64,ZGF0YTphcHBsaWNhdGlvbi9qc29uLHsiaW1hZ2VfZGF0YSI6ICIlM2NzdmclMjB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnJTIwdmlld0JveD0nMCUyMDAlMjAyNTUlMjAyNTUnJTIwd2lkdGg9JzUwMHB4JyUyMGhlaWdodD0nNTAwcHgnJTNlJTNjcGF0aCUyMGQ9J00wLDBIMjU1VjI1NUgwVjAnJTIwZmlsbD0nJTIzMUYyMTFEJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTAsMFYyNTVIMjU1QzI1NSwyNTUsMjE2LDQ3LDAsMCclMjBmaWxsPSclMjM3RUZBNTMnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTczLDIxN0g4MkM1NCwyMjQsNDgsMjU1LDQ4LDI1NUgyMDdDMjA3LDI1NSwyMDEsMjI0LDE3MywyMTcnJTIwZmlsbD0nJTIzNkY1OTk1JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTgyLDIxMUgxNzNWMjU1SDgyVjIxMSclMjBmaWxsPSclMjNGRUY3RUEnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNODMsMjExVjIyNUM5NiwyMzgsMTExLDI0NiwxMjcsMjQ2QzE0NCwyNDYsMTU5LDIzOCwxNzIsMjI1VjIxMUg4MyclMjBmaWxsPSclMjNERkQyQkMnLyUzZSUzY3BhdGglMjBkPSdNMTk3LDEwNkg1OEM0NiwxMDYsMzYsMTE3LDM2LDEzMUMzNiwxNDUsNDYsMTU2LDU4LDE1NkgxOTdDMjA5LDE1NiwyMTksMTQ1LDIxOSwxMzFDMjE5LDExNywyMDksMTA2LDE5NywxMDYnJTIwZmlsbD0nJTIzQUNBNzlEJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTU4LDEzNUE2OSw5OSwwLDEsMCwxOTcsMTM1QTY5LDk5LDAsMSwwLDU4LDEzNSclMjBmaWxsPSclMjNGRUY3RUEnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTk2LDEyMEMxOTYsMTIwLDIwOSwxMTgsMjEwLDEzNSclMjBmaWxsPSdub25lJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTU5LDEyMEM1OSwxMjAsNDYsMTE4LDQ1LDEzNSclMjBmaWxsPSdub25lJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTc5LDEzNUM3OSw4NiwxMDQsNDUsMTM2LDM4QTQ0LDQ0LDAsMCwwLDEyNiwzN0M4OSwzNyw1OSw4MSw1OSwxMzVDNTksMTkwLDg5LDIzMywxMjYsMjMzQTUwLDUwLDAsMCwwLDEzNiwyMzJDMTA0LDIyNSw3OSwxODUsNzksMTM1JyUyMGZpbGw9JyUyM0RGRDJCQycvJTNlJTNjcGF0aCUyMGQ9J00xMjcsMTkwQzk3LDE5MCw5MCwxNzksOTAsMTc5QzkwLDE3OSwxMDEsMjEyLDEyNywyMTJDMTU0LDIxMiwxNjUsMTc5LDE2NSwxNzlDMTY1LDE3OSwxNTgsMTkwLDEyNywxOTAnJTIwZmlsbD0nJTIzRDI3MUZGJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEyNywxOTdDMTExLDE5NywxMDcsMTkyLDEwNywxOTJDMTA3LDE5MiwxMTMsMjA2LDEyNywyMDZDMTQyLDIwNiwxNDgsMTkyLDE0OCwxOTJDMTQ4LDE5MiwxNDQsMTk3LDEyNywxOTcnJTIwZmlsbD0nJTIzMTcxNzE3JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEzMCwxOTFBODMsODMsMCwwLDEsMTIwLDE5MEE3OSw3OSwwLDAsMCwxNDAsMTkyQTc3LDc3LDAsMCwwLDE2MCwxOTBBNTYsNTYsMCwwLDAsMTY0LDE4MUMxNjQsMTgxLDE1NywxOTEsMTMwLDE5MSclMjBmaWxsPSclMjM4QjIzRDgnLyUzZSUzY3BhdGglMjBkPSdNMTAzLDE4OEM5NCwxODUsOTIsMTgxLDkyLDE4MUM5MiwxODEsMTAxLDIxMCwxMjUsMjEwVjIxMEMxMTMsMjA2LDEwNiwxOTQsMTAzLDE4OCclMjBmaWxsPSclMjM4QjIzRDgnLyUzZSUzY3BhdGglMjBkPSdNMTMxLDE3NEgxMjNBNCw0LDAsMCwwLDExOSwxNzlWMTc5QTQsNCwwLDAsMCwxMjMsMTg0SDEzMUE0LDQsMCwwLDAsMTM2LDE3OVYxNzlBNCw0LDAsMCwwLDEzMSwxNzQnJTIwZmlsbD0nJTIzMUUxRTFFJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEwMSwxMjJBMTUsMTUsMCwwLDAsODYsMTM3QTE3LDE3LDAsMCwwLDg3LDE0MkExNSwxNSwwLDAsMSwxMTYsMTQyQTE3LDE3LDAsMCwwLDExNywxMzdBMTUsMTUsMCwwLDAsMTAxLDEyMiclMjBmaWxsPSclMjMwMDAwMDAnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNODcsMTQyQTE0LDEwLDAsMSwwLDExNiwxNDJBMTQsMTAsMCwxLDAsODcsMTQyJyUyMGZpbGw9JyUyM0ZGRkZGRiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMDEsMTM3QTQsNCwwLDEsMCwxMDYsMTQyQTQsNCwwLDAsMCwxMDEsMTM3JyUyMGZpbGw9JyUyMzAwMDAwMCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xNTQsMTIyQTE1LDE1LDAsMCwwLDEzOCwxMzdBMTQsMTQsMCwwLDAsMTM5LDE0MkExNSwxNSwwLDAsMSwxNjgsMTQyQTE0LDE0LDAsMCwwLDE2OSwxMzdBMTUsMTUsMCwwLDAsMTU0LDEyMiclMjBmaWxsPSclMjMwMDAwMDAnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTM5LDE0MkExNCwxMCwwLDEsMCwxNjgsMTQyQTE0LDEwLDAsMSwwLDEzOSwxNDInJTIwZmlsbD0nJTIzRkZGRkZGJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE1NCwxMzdBNCw0LDAsMSwwLDE1OCwxNDJBNCw0LDAsMCwwLDE1NCwxMzcnJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEwOSwxMDlIOTRBNCw0LDAsMSwwLDk0LDExN0gxMDlBNCw0LDAsMCwwLDEwOSwxMDlWMTA5JyUyMGZpbGw9JyUyMzAwMDAwMCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xNjEsMTA5SDE0NkE0LDQsMCwxLDAsMTQ2LDExN0gxNjFBNCw0LDAsMSwwLDE2MSwxMDlWMTA5JyUyMGZpbGw9JyUyMzAwMDAwMCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00yMDIsMTUzQTUsNSwwLDAsMCwxOTcsMTU5QTUsNSwwLDEsMCwyMDIsMTUzJyUyMGZpbGw9JyUyM0FCQUJBQiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J001MywxNTNBNSw1LDAsMCwwLDQ3LDE1OUE1LDUsMCwxLDAsNTMsMTUzJyUyMGZpbGw9JyUyM0FCQUJBQiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00yMDMsMTU0QTUsNSwwLDAsMCwxOTksMTU1VjE1NUE1LDUsMCwwLDEsMjA0LDE2MEE0LDQsMCwwLDEsMjAzLDE2NEE1LDUsMCwxLDAsMjAzLDE1NFYxNTQnJTIwZmlsbD0nJTIzNzc3Nzc3Jy8lM2UlM2NwYXRoJTIwZD0nTTQ3LDE1OUE1LDUsMCwwLDAsNTIsMTY0QTQsNCwwLDAsMSw1MSwxNjBBNSw1LDAsMCwxLDU2LDE1NVYxNTVBNCw0LDAsMCwwLDUyLDE1NEE1LDUsMCwwLDAsNDcsMTU5JyUyMGZpbGw9JyUyMzc3Nzc3NycvJTNlJTNjcGF0aCUyMGQ9J00xNjUsMTg3QTUzLDUzLDAsMCwwLDE0MSwxNjJBMjgsMjgsMCwwLDAsMTE0LDE2MkE1NCw1NCwwLDAsMCw5MCwxODdDNzksMjE0LDEwNiwyMzUsMTI3LDIzNUMxNDksMjM1LDE3NiwyMTQsMTY1LDE4NyclMjBmaWxsPSclMjNDRkNGQ0YnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNNTgsMTMzVjEzNUM1OCwxMzgsNTgsMTQyLDU4LDE0NUM2NCwxNjAsODMsMTcyLDk1LDE3OEE1NCw1NCwwLDAsMSwxMDIsMTcxQzg5LDE2Niw2NiwxNTMsNTgsMTMzJyUyMGZpbGw9JyUyMzFDMUMxQyclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xOTcsMTMzQzE4OSwxNTMsMTY2LDE2NiwxNTMsMTcxQTUzLDUzLDAsMCwxLDE1OSwxNzhDMTcyLDE3MiwxOTEsMTYwLDE5NywxNDVDMTk3LDE0MiwxOTcsMTM4LDE5NywxMzVDMTk3LDEzNCwxOTcsMTMzLDE5NywxMzMnJTIwZmlsbD0nJTIzMUMxQzFDJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE0MSwxNjJBMjgsMjgsMCwwLDAsMTE0LDE2MkE1NCw1NCwwLDAsMCwxMDYsMTY3TDEyMywxOTFBNSw1LDAsMCwwLDEzMiwxOTFMMTQ5LDE2N0E1MCw1MCwwLDAsMCwxNDEsMTYyJyUyMGZpbGw9JyUyM0UxRTFFMSclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMDcsMTY2SDEwOEgxMDcnJTIwZmlsbD0nJTIzOTI5MjkyJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEwOCwxNjZMMTA5LDE2NEwxMDgsMTY2JyUyMGZpbGw9JyUyMzkyOTI5MiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMjMsMTkxTDEwNiwxNjdMMTA3LDE2NkE1Myw1MywwLDAsMCw5MCwxODdDNzksMjE0LDEwNiwyMzUsMTI3LDIzNVYxOTNBNSw1LDAsMCwxLDEyMywxOTEnJTIwZmlsbD0nJTIzOTI5MjkyJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEwNywxNjdMMTE1LDE3OEMxMTUsMTc4LDEyOSwxNjIsMTQ1LDE2NUMxNDUsMTY1LDEyNywxNTAsMTA3LDE2NyclMjBmaWxsPSclMjNGRkZGRkYnLyUzZSUzY3BhdGglMjBkPSdNMTEzLDE3OEwxMDYsMTY5Qzc0LDE5Niw5MywyMjksMTE4LDIzMEM4OCwyMDQsMTEzLDE3OCwxMTMsMTc4JyUyMGZpbGw9JyUyM0I3QjdCNycvJTNlJTNjcGF0aCUyMGQ9J003MywyMDlMNzQsMjExQTIzLDIzLDAsMCwxLDczLDIwOSclMjBmaWxsPSclMjM1OUFEM0InJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNNzIsMjA4TDczLDIwOUw3MiwyMDgnJTIwZmlsbD0nJTIzNTlBRDNCJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE4Niw5NlY5N0w4Nyw5M0w2Myw5Nkw2NCw5N0MyNiwxMDQsNjUsMTkyLDcyLDIwOEM2MSwxODIsNjksMTcwLDczLDE0MkM3NSwxMjIsNzIsMTA3LDcwLDEwMUM3NSwxMDQsODEsMTA4LDgzLDExM0M4MywxMTMsODQsMTA5LDgzLDEwNkM4MywxMDYsMTE1LDEwMywxMzIsMTEzQzEzMiwxMTMsMTMwLDEwNiwxMjYsMTA1QzEyNiwxMDUsMTcwLDEwNCwxNzQsMTEzQzE3NCwxMTMsMTcxLDEwMCwxODYsOTdDMTg0LDEwMSwxNzksMTE4LDE4MiwxNDJDMTg2LDE3MCwxOTQsMTgyLDE4MywyMDhDMTkwLDE5MSwyMzIsOTYsMTg2LDk2JyUyMGZpbGw9JyUyMzU5QUQzQiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xODIsMjA5QTIzLDIzLDAsMCwxLDE4MSwyMTFMMTgyLDIwOSclMjBmaWxsPSclMjM1OUFEM0InJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTgzLDIwOEwxODIsMjA5TDE4MywyMDgnJTIwZmlsbD0nJTIzNTlBRDNCJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTY4LDk3Vjk3SDY3SDY1Vjk3QzYwLDk4LDU1LDEwMCw1MiwxMDVDNDAsMTI0LDU3LDE3MSw2NywxOTVBNTUsNTUsMCwwLDEsNjYsMTc2QzU5LDE1NCw1MiwxMjUsNjAsMTExQTE3LDE3LDAsMCwxLDcwLDEwMkM3MCwxMDEsNjksMTAwLDY5LDk5Qzc0LDEwMiw4MCwxMDcsODMsMTExQTE1LDE1LDAsMCwwLDgyLDEwN0E3Niw3NiwwLDAsMCw2OCw5NyclMjBmaWxsPSclMjMzMTY2MUInLyUzZSUzY3BhdGglMjBkPSdNMjAzLDEwNUMxOTksMTAwLDE5NCw5NywxODcsOTdDMTg2LDk4LDE4Niw5OSwxODUsMTAyQTE3LDE3LDAsMCwxLDE5NSwxMTBDMjAzLDEyNSwxOTYsMTU0LDE4OSwxNzZBNTcsNTcsMCwwLDEsMTg4LDE5NUMxOTgsMTcxLDIxNSwxMjQsMjAzLDEwNSclMjBmaWxsPSclMjMzMTY2MUInLyUzZSUzY3BhdGglMjBkPSdNNTUsNzNIMjAwVjEwMUg1NVY3MyclMjBmaWxsPSclMjM2RDFFQUQnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTI3LDE5Qzc4LDE5LDY4LDQ5LDY4LDU3VjczSDE4N1Y1N0MxODcsNDksMTc3LDE5LDEyNywxOSclMjBmaWxsPSclMjM4NjI4Q0EnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTI3LDE5QzgwLDE5LDY5LDQ4LDY5LDU3VjczSDg5VjU3Qzg5LDQ5LDk3LDI1LDEzMywxOUgxMjcnJTIwZmlsbD0nJTIzNTkxMTdGJy8lM2UlM2NwYXRoJTIwZD0nTTE4Niw3NEg2OUExMywxMywwLDAsMCw1Niw4N1Y4N0ExMywxMywwLDAsMCw2OSwxMDBWMTAwQTE0LDE0LDAsMCwxLDY3LDkyVjkyQTEzLDEzLDAsMCwxLDgxLDc4SDE5NkExMiwxMiwwLDAsMCwxODYsNzQnJTIwZmlsbD0nJTIzNDYwRDZCJy8lM2UlM2NzdHlsZSUzZXBhdGh7c3Ryb2tlLXdpZHRoOjAuNzF9JTNjL3N0eWxlJTNlJTNjL3N2ZyUzZSIsICJuYW1lIjogIkRyZWFtZXIlMjAlMjM3IiwgImRlc2NyaXB0aW9uIjogIlJ1bm5lcnMlMjBydW4sJTIwYnV0JTIwc29tZXRpbWVzJTIwdGhleSUyMGRyZWFtLiUyMFRoaXMlMjBpcyUyMG9uZSUyMG9mJTIwdGhlaXIlMjBkcmVhbXMuIiwiYXR0cmlidXRlcyI6ICIifQ==