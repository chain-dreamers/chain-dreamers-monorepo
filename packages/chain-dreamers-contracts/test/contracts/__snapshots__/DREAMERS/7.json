data:application/json,{"image_data": "%3csvg%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%20255%20255'%20width='500px'%20height='500px'%3e%3cpath%20d='M0,0H255V255H0V0'%20fill='%231F211D'%20stroke='%23000'/%3e%3cpath%20d='M0,0V255H255C255,255,216,47,0,0'%20fill='%237EFA53'%20stroke='%23000'/%3e%3cpath%20d='M173,217H82C54,224,48,255,48,255H207C207,255,201,224,173,217'%20fill='%236F5995'%20stroke='%23000'/%3e%3cpath%20d='M82,211H173V255H82V211'%20fill='%23FEF7EA'%20stroke='%23000'/%3e%3cpath%20d='M83,211V225C96,238,111,246,127,246C144,246,159,238,172,225V211H83'%20fill='%23DFD2BC'/%3e%3cpath%20d='M197,106H58C46,106,36,117,36,131C36,145,46,156,58,156H197C209,156,219,145,219,131C219,117,209,106,197,106'%20fill='%23ACA79D'%20stroke='%23000'/%3e%3cpath%20d='M58,135A69,99,0,1,0,197,135A69,99,0,1,0,58,135'%20fill='%23FEF7EA'%20stroke='%23000'/%3e%3cpath%20d='M196,120C196,120,209,118,210,135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M59,120C59,120,46,118,45,135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M79,135C79,86,104,45,136,38A44,44,0,0,0,126,37C89,37,59,81,59,135C59,190,89,233,126,233A50,50,0,0,0,136,232C104,225,79,185,79,135'%20fill='%23DFD2BC'/%3e%3cpath%20d='M127,190C97,190,90,179,90,179C90,179,101,212,127,212C154,212,165,179,165,179C165,179,158,190,127,190'%20fill='%23D271FF'%20stroke='%23000'/%3e%3cpath%20d='M127,197C111,197,107,192,107,192C107,192,113,206,127,206C142,206,148,192,148,192C148,192,144,197,127,197'%20fill='%23171717'%20stroke='%23000'/%3e%3cpath%20d='M130,191A83,83,0,0,1,120,190A79,79,0,0,0,140,192A77,77,0,0,0,160,190A56,56,0,0,0,164,181C164,181,157,191,130,191'%20fill='%238B23D8'/%3e%3cpath%20d='M103,188C94,185,92,181,92,181C92,181,101,210,125,210V210C113,206,106,194,103,188'%20fill='%238B23D8'/%3e%3cpath%20d='M131,174H123A4,4,0,0,0,119,179V179A4,4,0,0,0,123,184H131A4,4,0,0,0,136,179V179A4,4,0,0,0,131,174'%20fill='%231E1E1E'%20stroke='%23000'/%3e%3cpath%20d='M101,122A15,15,0,0,0,86,137A17,17,0,0,0,87,142A15,15,0,0,1,116,142A17,17,0,0,0,117,137A15,15,0,0,0,101,122'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M87,142A14,10,0,1,0,116,142A14,10,0,1,0,87,142'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M101,137A4,4,0,1,0,106,142A4,4,0,0,0,101,137'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M154,122A15,15,0,0,0,138,137A14,14,0,0,0,139,142A15,15,0,0,1,168,142A14,14,0,0,0,169,137A15,15,0,0,0,154,122'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M139,142A14,10,0,1,0,168,142A14,10,0,1,0,139,142'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M154,137A4,4,0,1,0,158,142A4,4,0,0,0,154,137'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M109,109H94A4,4,0,1,0,94,117H109A4,4,0,0,0,109,109V109'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M161,109H146A4,4,0,1,0,146,117H161A4,4,0,1,0,161,109V109'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M202,153A5,5,0,0,0,197,159A5,5,0,1,0,202,153'%20fill='%23ABABAB'%20stroke='%23000'/%3e%3cpath%20d='M53,153A5,5,0,0,0,47,159A5,5,0,1,0,53,153'%20fill='%23ABABAB'%20stroke='%23000'/%3e%3cpath%20d='M203,154A5,5,0,0,0,199,155V155A5,5,0,0,1,204,160A4,4,0,0,1,203,164A5,5,0,1,0,203,154V154'%20fill='%23777777'/%3e%3cpath%20d='M47,159A5,5,0,0,0,52,164A4,4,0,0,1,51,160A5,5,0,0,1,56,155V155A4,4,0,0,0,52,154A5,5,0,0,0,47,159'%20fill='%23777777'/%3e%3cpath%20d='M165,187A53,53,0,0,0,141,162A28,28,0,0,0,114,162A54,54,0,0,0,90,187C79,214,106,235,127,235C149,235,176,214,165,187'%20fill='%23CFCFCF'%20stroke='%23000'/%3e%3cpath%20d='M58,133V135C58,138,58,142,58,145C64,160,83,172,95,178A54,54,0,0,1,102,171C89,166,66,153,58,133'%20fill='%231C1C1C'%20stroke='%23000'/%3e%3cpath%20d='M197,133C189,153,166,166,153,171A53,53,0,0,1,159,178C172,172,191,160,197,145C197,142,197,138,197,135C197,134,197,133,197,133'%20fill='%231C1C1C'%20stroke='%23000'/%3e%3cpath%20d='M141,162A28,28,0,0,0,114,162A54,54,0,0,0,106,167L123,191A5,5,0,0,0,132,191L149,167A50,50,0,0,0,141,162'%20fill='%23E1E1E1'%20stroke='%23000'/%3e%3cpath%20d='M107,166H108H107'%20fill='%23929292'%20stroke='%23000'/%3e%3cpath%20d='M108,166L109,164L108,166'%20fill='%23929292'%20stroke='%23000'/%3e%3cpath%20d='M123,191L106,167L107,166A53,53,0,0,0,90,187C79,214,106,235,127,235V193A5,5,0,0,1,123,191'%20fill='%23929292'%20stroke='%23000'/%3e%3cpath%20d='M107,167L115,178C115,178,129,162,145,165C145,165,127,150,107,167'%20fill='%23FFFFFF'/%3e%3cpath%20d='M113,178L106,169C74,196,93,229,118,230C88,204,113,178,113,178'%20fill='%23B7B7B7'/%3e%3cpath%20d='M73,209L74,211A23,23,0,0,1,73,209'%20fill='%2359AD3B'%20stroke='%23000'/%3e%3cpath%20d='M72,208L73,209L72,208'%20fill='%2359AD3B'%20stroke='%23000'/%3e%3cpath%20d='M186,96V97L87,93L63,96L64,97C26,104,65,192,72,208C61,182,69,170,73,142C75,122,72,107,70,101C75,104,81,108,83,113C83,113,84,109,83,106C83,106,115,103,132,113C132,113,130,106,126,105C126,105,170,104,174,113C174,113,171,100,186,97C184,101,179,118,182,142C186,170,194,182,183,208C190,191,232,96,186,96'%20fill='%2359AD3B'%20stroke='%23000'/%3e%3cpath%20d='M182,209A23,23,0,0,1,181,211L182,209'%20fill='%2359AD3B'%20stroke='%23000'/%3e%3cpath%20d='M183,208L182,209L183,208'%20fill='%2359AD3B'%20stroke='%23000'/%3e%3cpath%20d='M68,97V97H67H65V97C60,98,55,100,52,105C40,124,57,171,67,195A55,55,0,0,1,66,176C59,154,52,125,60,111A17,17,0,0,1,70,102C70,101,69,100,69,99C74,102,80,107,83,111A15,15,0,0,0,82,107A76,76,0,0,0,68,97'%20fill='%2331661B'/%3e%3cpath%20d='M203,105C199,100,194,97,187,97C186,98,186,99,185,102A17,17,0,0,1,195,110C203,125,196,154,189,176A57,57,0,0,1,188,195C198,171,215,124,203,105'%20fill='%2331661B'/%3e%3cpath%20d='M55,73H200V101H55V73'%20fill='%236D1EAD'%20stroke='%23000'/%3e%3cpath%20d='M127,19C78,19,68,49,68,57V73H187V57C187,49,177,19,127,19'%20fill='%238628CA'%20stroke='%23000'/%3e%3cpath%20d='M127,19C80,19,69,48,69,57V73H89V57C89,49,97,25,133,19H127'%20fill='%2359117F'/%3e%3cpath%20d='M186,74H69A13,13,0,0,0,56,87V87A13,13,0,0,0,69,100V100A14,14,0,0,1,67,92V92A13,13,0,0,1,81,78H196A12,12,0,0,0,186,74'%20fill='%23460D6B'/%3e%3cstyle%3epath{stroke-width:0.71}%3c/style%3e%3c/svg%3e", "name": "Dreamer%20%237", "description": "Runners%20run,%20but%20sometimes%20they%20dream.%20This%20is%20one%20of%20their%20dreams.","attributes": ""}